### PR TITLE
POPR-180: Configure AWS amplify deploy investigation

### PR DESCRIPTION
**Pull request by AgusPk**
# Related Task
- https://app.clickup.com/t/459857/POPR-180
- 
# Proposed Changes
adjustment to webpack config

# Task Checklist
- [x] All new features have been documented
- [x] Conventions and Acceptance criteria have been met
- [x] Code is free of warnings
- [x] ClickUp task has been set to Awaiting Review

### CODE REVIEW

File: frontend/app/config/webpack.production.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ const config = {
       PUBLIC_URL,
     }),
     new Dotenv({
-      path: commonPaths.prodEnv,
+      systemvars: true,
```
Comment (pamcontin) [inline frontend/app/config/webpack.production.js:49]: Just so to now, what does this line do?

**Review by pamcontin — APPROVED** [review]
Everything looks great! :rocket:

File: frontend/app/config/webpack.production.js (hunk 1 above)
Comment (AgusPk) [reply frontend/app/config/webpack.production.js:49]: that plugin filters the env variables for default, if you set systemvars true, you allow variables defined in other parts of the system to be read, for example, the ones you set in amplify

Comment (pamcontin) [reply frontend/app/config/webpack.production.js:49]: cool! thenks!!

**Pull request merged**